Octokit/Http/JsonHttpPipeline.cs

```diff
@@ -30,7 +33,8 @@ public void SerializeRequest(IRequest request)
 
             if (!request.Headers.ContainsKey("Accept"))
             {
-                request.Headers["Accept"] = "application/vnd.github.v3+json; charset=utf-8";
+
+                request.Headers["Accept"] = String.Join(",", moondragonPreviewApiVersion, v3ApiVersion);
```

Comment:
@joshvera my testing of this (and a part of the HTTP spec I neglected to recall) indicates that the first match of an `Accept` value is the one the server will use. In this case, we'd need to put the preview API first.

![](http://www.billboard.com/files/styles/promo_650/public/media/katy-perry-the-more-you-know-super-bowl-half-time-2015-billboard-650.jpg)